itest/starboard_cli_test.go

```diff
@@ -64,6 +64,91 @@ func NewPodSpec(podName string, containers map[string]string, args ...string) (*
 	return pod, err
 }
 
+var _ = Describe("Starboard CLI dryrun", func() {
+
+	Describe("Command init with dryrun", func() {
+
+		It("should return an error if the wrong dryrun flag is provided", func() {
+			out := NewBuffer()
+			err := cmd.Run(versionInfo, []string{
+				"starboard",
+				"init",
+				"--dry",
+				"-v", starboardCLILogLevel,
+			}, out, out)
+			Expect(err).To(HaveOccurred())
```

Comment:
Did you mean testing a wrong flag (--dry instead of --dry-run) or the wrong value of the --dry-run flag?

I think in this case we'll get the following error: `error: unknown flag: --dry`

Reply:
Yes, I was testing an invalid option... which should return an error as you have noted.